test/powershell/engine/Help/assets/Microsoft.PowerShell.LocalAccounts_8e362604-2c0b-448f-a414-a6a690a644e2_HelpInfo.xml

```diff
@@ -1,10 +1,10 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <HelpInfo xmlns="http://schemas.microsoft.com/powershell/help/2010/05">
-  <HelpContentURI>http://go.microsoft.com/fwlink/?LinkId=717973</HelpContentURI>
+  <HelpContentURI>https://go.microsoft.com/fwlink/?LinkId=717973</HelpContentURI>
```

Comment:
It works?
Should we change previous line too?

Reply:
302 redirect to http://download.microsoft.com/download/2/3/0/23073AB1-2A89-45F0-8673-FBBA4636AF06/
and then 403 (forbidden)

The previous line refers to a XML namespace not a URL so should be be changed.

Reply:
Issue created: #8690